### DISCUSSION TITLE
Netflow v9 packet rejected cause no default sampling rate. Fw : Palo alto

### DISCUSSION THREAD

**Discussion by Julien33x** [akvorado/akvorado#1453] [Q&A]
Hi Vincent, 

First I love the tool and I'm glad to deploy it. 

I need help about the option : default-sampling-rate. 

By default the flow was rejected and thanks to your documentation, I found why. 

But there's no really example or advice to set this option. 

I successfully set it to 5 and now I see my exporter on akvorado but it will be great to get some more information, recommendations and advices to tune this correctly. 

The fw is a panos, with template refresh rate setting but it seems that default # values are satisfying akvorado

From Palo I have :
Minutes :1
Packets : 200
Active time out (min) :5 

Many thanks for all the help that community can give me. 

Many thanks for this tools.

**Comment by Julien33x** [comment]
> Hi Vincent, 
> 
> First I love the tool and I'm glad to deploy it. 
> 
> I need help about the option : default-sampling-rate. 
> 
> By default the flow was rejected and thanks to your documentation, I found why. 
> 
> But there's no really example or advice to set this option. 
> 
> I successfully set it to 5 and now I see my exporter on akvorado but it will be great to get some more information, recommendations and advices to tune this correctly. 
> 
> The fw is a panos, with template refresh rate setting but it seems that default # values are satisfying akvorado
> 
> From Palo I have :
> Minutes :1
> Packets : 200
> Active time out (min) :5 
> 
> 
> Many thanks for all the help that community can give me. 
> 
> Many thanks for this tools. 
> 
> 

I found some articles about netflow and sampling and I am still searching the best way to handle the trouble for Palo alto netflow.

However you mentionned in the documentation that it can be set for a specific exporter.

Maybe this way will be a better way to handle that?

Could you please give me some advices?

**Comment by vincentbernat** [comment]
`default-sampling-rate` can be a map from IP to sampling rates:

```yaml
default-sampling-rate:
  192.168.1.3: 1
  10.0.0.0/8: 200
```

Either try 1 or 200 and check if you get the expected values in bps.
